react-toggle-switch-component/my-app/src/App.tsx

```diff
@@ -0,0 +1,11 @@
+import { ToggleSwitch } from './ToggleSwitch';
+
+function App() {
+  return (
+    <>
+      <ToggleSwitch text="OFF" className="off" />
```

Comment:
I'm not sure I understand why you are passing in these props?

It's best if your components are self-contained and don't require the parent to know anything about how the component works. Here you are forcing the parent to know that it uses the class "off" and sets the text to OFF.